src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsLanguageDebugInfo.cs

```diff
@@ -149,13 +153,17 @@ public int GetProximityExpressions(IVsTextBuffer pBuffer, int iLine, int iCol, i
                             Document document = snapshot.GetOpenDocumentInCurrentContextWithChanges();
                             if (document != null)
                             {
-                                var point = snapshot.GetPoint(iLine, iCol);
-                                var proximityExpressions = _proximityExpressionsService.GetProximityExpressionsAsync(document, point.Position, waitContext.CancellationToken).WaitAndGetResult(waitContext.CancellationToken);
-
-                                if (proximityExpressions != null)
+                                var nullablePoint = snapshot.TryGetPoint(iLine, iCol);
+                                if (nullablePoint.HasValue)
                                 {
-                                    enumBSTR = new VsEnumBSTR(proximityExpressions);
-                                    succeeded = true;
+                                    var point = nullablePoint.Value;
+                                    var proximityExpressions = _proximityExpressionsService.GetProximityExpressionsAsync(document, point.Position, waitContext.CancellationToken).WaitAndGetResult(waitContext.CancellationToken);
+
+                                    if (proximityExpressions != null)
+                                    {
+                                        enumBSTR = new VsEnumBSTR(proximityExpressions);
+                                        succeeded = true;
+                                    }
                                 }
                             }
                         }
```

Comment:
The same here re unit testing.

Reply:
agreed to take this after 15.7
added [https://github.com/dotnet/roslyn/issues/26133](https://github.com/dotnet/roslyn/issues/26133)

---
In reply to: [181254772](https://github.com/dotnet/roslyn/pull/26128#discussion_r181254772) [](ancestors = 181254772)